src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayCount = days;
+  const totalPay = days * 40;
+
+  if (dayCount >= 3 && dayCount < 7) {
+    return totalPay - 20;
+  } else if (dayCount >= 7) {
+    return totalPay - 50;
+  }
```

Comment:
Our mentors don't like Harry Potter & hate magic (
Move this "magic numbers" ( 20$ and 50$) to variables

Reply:
okey, thanks)